src/main/java/core/basesyntax/RightTriangleSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class RightTriangleSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();
+    private static final ColorSupplier color = new ColorSupplier();

```

Comment:
The instance variable 'color' should not be static. Make it a non-static field or move it into the method.